src/realm/array_with_find.hpp

```diff
@@ -0,0 +1,1064 @@
+/*************************************************************************
```

Comment:
I am assuming this is all purely moved code that I am not going to review in depth.

Reply:
It is.